src/nora/components/NoraCheckboxInput/NoraCheckboxInput.md

```diff
@@ -23,3 +23,50 @@ that accounts for the generally smaller font sizes we use in Nora.
   </a>
 </NoraCheckboxInput>
 ```
+
+Example with one checkbox setting the value of another
+
+```jsx
+import React, { useState } from 'react'
+
+function NoraCheckboxExample() {
+  const [labSelected, setLabSelected] = useState(true)
+  const [ceaSelected, setCeaSelected] = useState(false)
+
+  // Instead of a validator setting the state of the labs, we can also consider disabling the checkbox if one of the children are selected
+  return (
+    <>
+      <NoraCheckboxInput
+        name="labs"
+        data-tid="labs-checkbox"
+        checked={labSelected}
+        validator={(val) => {
+          setLabSelected(val)
+          if (!val) {
```

Comment:
Checking against `val` and not `labSelected` and `ceaSelected` was also key as it was causing a race condition.